ImageViewer/Source/GalleryViewController.swift

```diff
@@ -396,13 +407,17 @@ open class GalleryViewController: UIPageViewController, ItemControllerDelegate {
 
         let thumbnailsController = ThumbnailsViewController(itemsDataSource: self.itemsDataSource)
 
-        if let closeButton = closeButton {
+        if let closeButton = seeAllCloseButton {
+            thumbnailsController.closeButton = closeButton
+            thumbnailsController.closeLayout = seeAllCloseLayout
+        } else if let closeButton = closeButton {
             let seeAllCloseButton = UIButton(frame: CGRect(origin: CGPoint.zero, size: closeButton.bounds.size))
             seeAllCloseButton.setImage(closeButton.image(for: UIControlState()), for: UIControlState())
```

Comment:
not related to this PR but this should be `.normal` instead of `UIControlState()`... will fix it at some point